tokens/index.js

```diff
@@ -4,17 +4,27 @@ import fs from 'node:fs';
 import path from 'node:path';
 import { minify } from '../css-minify.js';
 import drnm from 'drnm';
+import postcss from 'postcss';
+import atImport from 'postcss-import';
 
 const __dirname = drnm(import.meta.url);
 const outPath = path.join(__dirname, '../dist/tokens');
 fs.mkdirSync(outPath, { recursive: true });
 
-function process(tld) {
+async function process(tld) {
+  const slugifiedName = slugify(tld);
+  const filename = path.join(outPath, slugifiedName) + '.css';
+
+  const fontDefinitions = `@import '@warp-ds/fonts/dist/${path.basename(filename)}';`;
   const tokens = tokenize(`./${tld}`);
-  const css = minify(tokens);
-  const filename = path.join(outPath, slugify(tld)) + '.css';
-  const fontsImport = `@import'https://assets.finn.no/pkg/@warp-ds/fonts/v1/${path.basename(filename)}';`;
-  fs.writeFileSync(filename, `${fontsImport}${css}`, 'utf-8');
+  const css = `${fontDefinitions};${tokens}`;
```

Comment:
While testing I couldn't see a `;` between tokens and font definitions 🤔 
But maybe it is ok and is removed by a minifier

<img width="881" alt="image" src="https://github.com/warp-ds/css/assets/37986637/3c5fccf9-377f-40bb-a92f-f0655decb229">

Reply:
It got me at first, too! But `;` is used at the end of each line within `{}`, so this is correct. You can also see some examples on [MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) and there's no semi-colon after `{}` either 😄